### PR TITLE
Better descriptor of what the "Keyboard shortcut" shortcut does

### PR DESCRIPTION
**Pull request by N6REJ**
Pull Request for Issue # .

### Summary of Changes
Changes "Keyboard Shortcuts" text shown at bottom of admin to "Show keyboard shortcuts"

### Testing Instructions
Check bottom of admin panel showing the "Keyboard Shortcuts"
apply patch
Check that it has changed.

### Actual result BEFORE applying this Pull Request

image

### Expected result AFTER applying this Pull Request

image

### Documentation Changes Required

if used then Update user documentation showing the change in verbiage.

### CODE REVIEW

**Comment by brianteeman** [comment]
Less is more - this change adds nothing

**Comment by N6REJ** [comment]
> Less is more - this change adds nothing

Your welcome to your opinion.

**Comment by brianteeman** [comment]
Not just mine

github
image

gmail
image

tweetdeck
image

facebook
image

**Comment by N6REJ** [comment]
if your going to post examples the be relevant and accurate.  Others ineptness at the english language is not my fault.
as FB clearly shows, their shorcut is labeled in such a way to in imply its going to a legend.  They simply chose different verbiage.
image

**Comment by N6REJ** [comment]
anyway, I'm not going to continue to have a battle of wits with an unarmed opponent.

**Comment by brianteeman** [comment]
> if your going to post examples the be relevant and accurate. Others ineptness at the English language is not my fault.

You submitted a pull request for review and I have offered my review. You didn't provide any evidence that it is better to be overly verbose or that users have been confused about what the expected behaviour is. It is well documented that less is more when it comes to onscreen information and it's included in the style guide. Not every proposed change is an improvement and we all see contributions rejected at some time or other.

There is no need for personal attacks.

**Comment by mrownicki** [comment]
Btw. 
The description of J + X is misleading. Because suggest using click together. Should be J -> X or J and X (not together).<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38755">issues.joomla.org/tracker/joomla-cms/38755</a>.</sub>

**Comment by brianteeman** [comment]
There is a pull request already for that

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 203f8e4c07a22147592b12b1240c81405c2dfc07<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38755">issues.joomla.org/tracker/joomla-cms/38755</a>.</sub>

**Comment by chmst** [comment]
I like the "Show keyboard shortcuts". 
But I thin the headline should be only "Joomla Short Cuts"

grafik

**Comment by brianteeman** [comment]
please update the original post

please use capitalisation on the title<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38755">issues.joomla.org/tracker/joomla-cms/38755</a>.</sub>

**Comment by N6REJ** [comment]
> please update the original post
> 
> please use capitalisation on the title
> 
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/38755](https://issues.joomla.org/tracker/joomla-cms/38755).

huh?

File: administrator/language/en-GB/plg_system_shortcut.ini [hunk 1]
```diff
@@ -3,10 +3,10 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-PLG_SYSTEM_SHORTCUT="System - Keyboard Shortcuts"
+PLG_SYSTEM_SHORTCUT="System - Show keyboard shortcuts"
```
Comment (Kostelano) [inline administrator/language/en-GB/plg_system_shortcut.ini]: Very stupid plugin name

Comment (brianteeman) [reply administrator/language/en-GB/plg_system_shortcut.ini]: it is just kplain wrong. The plugin does not show shortcuts. the pluin IS shortcuts

**Comment by N6REJ** [comment]
Everyone should be happy now.  It's been redone correctly.
Sorry I made mistakes in it before.

**Comment by N6REJ** [comment]
> I have tested this item ✅ successfully on [203f8e4](https://github.com/joomla/joomla-cms/commit/203f8e4c07a22147592b12b1240c81405c2dfc07)
> 
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/38755](https://issues.joomla.org/tracker/joomla-cms/38755).

needs retesting due to errors on my part

**Comment by brianteeman** [comment]
I assume that you will be updating h rest of jomla with this useless extra word eg Show Help, Show Options Show Articles etc

**Comment by brianteeman** [comment]
not  a personal comment its a statement of fact. If we add the useless word here then we have to also add the useless word in all similar use cases.

**Comment by N6REJ** [comment]
Well, being I'm not one of the dev's who created it, when I first saw it my thoughts were...
hmmm what in the world is this?
click
OH!.. why didn't it tell me that?
so, I'm probably not the only one who feels/thinks that way.

**Comment by N6REJ** [comment]
I give up.  It's ok for others to make changes but not me.

**Pull request closed without merging**